.github/CODEOWNERS

```diff
@@ -26,6 +26,9 @@
 # Airflow CTL
 /airflow-ctl/ @bugraoz93 @potiuk
 
+# Airflow CLI
+/airflow-core/src/airflow/cli/ @bugraoz93 @poituk
```

Comment:
```suggestion
/airflow-core/src/airflow/cli/ @bugraoz93 @potiuk 
```

Reply:
Oops, pressed tab for AI, which caused the trouble :sweat_smile: Thanks a lot, Vincent!